src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -46,6 +48,45 @@ var defaultStartManagementServerFunc startManagementServerFunc = func(s *service
 	shimLog.Info("management server started")
 }
 
+// calculateSandboxSizing will calculate the number of CPUs and amount of Memory that should
+// be added to the VM if sandbox annotations are provided with this sizing details
+func calculateSandboxSizing(annotations map[string]string) (numCPU, memSizeMB uint32) {
```

Comment:
While these aren't used anywhere else, I wonder if these functions would be better suited in a utility package. Having said that, I see that there are about 4 utility packages in the repo today, and which would be best isn't yet clear. I'm happy to take this on in follow up, or here if folks have opinions.

Reply:
Would it make sense to have them both take a `*specs.Spec` as an input and move them to `vc/pkg/oci`?

Reply:
Yes, agreed.